modules/kargoBidAdapter.js

```diff
@@ -284,6 +287,36 @@ export const spec = {
 
       triggerPixel(timeoutRequestUrl);
     } catch (e) {}
+  },
+
+  getImpression(bid) {
+   let imp = {
+      id: bid.bidId,
+      tid: bid.transactionId,
+      pid: bid.params.placementId,
+      code: bid.adUnitCode,
+      floor: bid.floor,
+      banner: bid.mediaTypes.banner,
+      video: bid.mediaTypes.video,
+      fpd: { 
+        gpid: bid.ortb2Imp.gpid
```

Comment:
this should be good, but i'm wondering should we support both based on prebid docs: https://docs.prebid.org/features/pbAdSlot.html